components/table/stories/table.stories.js

```diff
@@ -242,22 +426,58 @@ Quiet.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
+/**
+ * Tables can enable column sorting. The `aria-sort` attribute should be set on the `.spectrum-Table-headCell` element for the column that is currently sorted. `aria-sort` should not be set on more than one column at a time.
+ *
+ * When a column is sorted in ascending order, use the `SortUp` icon, as seen below. If sorted in descending order, use the `SortDown` icon. If the sort is undefined, use the more general `Sort` icon.
+ *
+ * Implementations will develop their own sorting functionality. Additionally, the `aria-sort` attribute should be removed from a column header when the column is not sorted.
+ */
+export const Sortable = Template.bind({});
+Sortable.args = {
+	...Default.args,
+	isSortable: true,
+	sortIcon: "Sort",
+};
+Sortable.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+Sortable.storyName = "Sortable columns";
+Sortable.tags = ["!dev"];
+
+/**
+ * Tables can have additional actions in the header cells that could trigger a menu. The content of the header cell is an [button](/docs/components-button--docs). Oftentimes, the menu is used to show users the available sorting options.
+ */
+export const WithMenuButton = Template.bind({});
+WithMenuButton.args = {
+	...Default.args,
+	hasMenu: true,
+};
```

Comment:
I was expecting this story to have a different icon (the chevron/disclosure); should it? Should it have an icon at the start and the end?

![image](https://github.com/user-attachments/assets/2c8f68cb-2844-4bc7-a05b-e8a526316327)

Reply:
oh how weird! The storybook link isn't rendering that icon? I get it locally, but hadn't checked the preview link. Let me look into this.
<img width="818" alt="Screenshot 2025-06-05 at 9 45 45 AM" src="https://github.com/user-attachments/assets/6bb8149f-3e4e-432a-9c85-bda103da3720" />

Reply:
I made some updates to the source path I was using with `mask-image`. It looks like it builds correctly now in [the preview](https://pr-3799--spectrum-css.netlify.app/?path=/docs/components-table--docs)! 🥳